source/app/components/Modal/Modal.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import PropTypes from "prop-types";
+import classNames from 'classnames';
+import css from "./Modal.scss";
+
+Modal.propTypes = {
+    children: PropTypes.node.isRequired,
+    onClose: PropTypes.func.isRequired,
+    show: PropTypes.bool.isRequired
+};
+export default function Modal({
+    children,
+    onClose,
+    show
+}) {
+    // console.log("show"+show);
+    const modalClassNames = classNames(css.modal)
+    if (!show) {
+    return null;
+    }
+    return show && (
+        <div className={css.modal} id="modal">
+            <div className={css.modal__content}>
+                <h4>Cannot upload your own documents- Demonstration Mode</h4>
```

Comment:
Can we pass the title in the Modal function?
This will make it easier to re-use this modal for future cases

Reply:
Done.